docs/services-developer-guide.md

```diff
@@ -450,34 +450,38 @@ Crossplane-specific tasks.
 
 crossplane-runtime provides the following `reconcile.Reconcilers`:
 
-* The [`resource.ManagedReconciler`] reconciles managed resources with external
-  systems by instantiating a client of the external API and using it to create,
-  update, or delete the external resource as necessary.
-* [`resource.ClaimSchedulingReconciler`] reconciles resource claims by
-  scheduling them to a resource class that matches their class selector labels
+* The [`managed.Reconciler`] reconciles managed resources with external systems
+  by instantiating a client of the external API and using it to create, update,
+  or delete the external resource as necessary.
+* [`claimscheduling.Reconciler`] reconciles resource claims by scheduling them
+  to a resource class that matches their class selector labels (if any).
+* [`claimdefaulting.Reconciler`] reconciles resource claims that omit their
+  class selector by defaulting them to a resource class annotated as the default
   (if any).
-* [`resource.ClaimDefaultingReconciler`] reconciles resource claims that omit
-  their class selector by defaulting them to a resource class annotated as the
-  default (if any).
-* [`resource.ClaimReconciler`] reconciles resource claims with managed resources
+* [`claimbinding.Reconciler`] reconciles resource claims with managed resources
   by either binding or dynamically provisioning and then binding them.
-* [`resource.SecretPropagatingReconciler`] reconciles secrets by propagating
-  their data to another secret. This controller is typically used to ensure
-  resource claim connection secrets remain in sync with the connection secrets
-  of their bound managed resources.
+* [`secret.NewReconciler`] reconciles secrets by propagating their data to
```

Comment:
Perhaps we should use the `foo.NewReconciler` variant across all of these examples, since there is no `secret.Reconciler`? Just a nitpick for symmetry - happy to stick with what we have too.

Reply:
I didn't like this as well, but didn't want to change the current pattern based on this one. I think going to `NewReconciler` in future would be great though!